packages/warehouses/src/warehouseClients/SnowflakeWarehouseClient.ts

```diff
@@ -223,6 +223,9 @@ export class SnowflakeWarehouseClient extends WarehouseBaseClient<CreateSnowflak
             credentials.authenticationType ===
             SnowflakeAuthenticationType.EXTERNAL_BROWSER
         ) {
+            // This can only be used on the CLI
+            // The backend does not support this authentication type
```

Comment:
This client is shared between the backend and the cli, so we can't remove this auth type. 

we don't need to enforce this for now